samples/snippets/storage_print_pubsub_bucket_notification.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+"""Sample that gets a notification configuration for a bucket.
+This sample is used on this page:
+    https://cloud.google.com/storage/docs/reporting-changes
+For more information, see README.md.
+"""
+
+# [START storage_print_pubsub_bucket_notification]
+from google.cloud import storage
+
+
+def print_pubsub_bucket_notification(bucket_name, notification_id):
```

Comment:
I think at one point it was customary to omit the function declaration in a sample, and just have the region tag select the contents of the function declaration. Is that not the case anymore?

Reply:
I'm seeing both patterns in python-docs-samples, and nothing explicitly mentioned in the [Authoring Guide](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md). Reading our internal go/code-snippets-style guide, the sample omits the function declaration. Per our discussion offline, this will be included as part of the repository-level change/decision and in a separate PR if needed.